scripts/sync_repo_files.sh

```diff
@@ -185,6 +185,9 @@ process_repo() {
   cd "${tmp_dir}/${org_repo}" || return 1
   git checkout -b "${branch}" || return 1
 
+  # If we need to add an Actions file this directory needs to be present.
+  mkdir -p "./.github/workflows"
```

Comment:
While not all repositories being synced need the Action (some aren't Go), creating the directory doesn't do anything if there isn't a file in it so we can just run the command anyway.